docs/ad/api.md

```diff
@@ -649,101 +830,278 @@ GET _opendistro/_anomaly_detection/stats/<stat>
 
 ```json
 {
-  "_nodes": {
-    "total": 3,
-    "successful": 3,
-    "failed": 0
+  "_nodes" : {
+    "total" : 3,
+    "successful" : 3,
+    "failed" : 0
   },
-  "cluster_name": "multi-node-run",
-  "anomaly_detectors_index_status": "green",
-  "detector_count": 1,
-  "models_checkpoint_index_status": "green",
-  "anomaly_results_index_status": "green",
-  "nodes": {
-    "IgWDUfzFRzW0FWAXM5FGJw": {
-      "ad_execute_request_count": 8,
-      "models": [
+  "cluster_name" : "multi-node-run",
+  "anomaly_detectors_index_status" : "green",
+  "detector_count" : 1,
+  "models_checkpoint_index_status" : "green",
+  "anomaly_results_index_status" : "green",
+  "nodes" : {
+    "IgWDUfzFRzW0FWAXM5FGJw" : {
+      "ad_execute_request_count" : 8,
+      "ad_execute_failure_count" : 7,
+      "models" : [
         {
-          "detector_id": "asbPyG4BlFrvFHzloDO-",
-          "model_type": "rcf",
-          "model_id": "asbPyG4BlFrvFHzloDO-_model_rcf_0"
+          "detector_id" : "asbPyG4BlFrvFHzloDO-",
+          "model_type" : "rcf",
+          "model_id" : "asbPyG4BlFrvFHzloDO-_model_rcf_0"
         },
         {
-          "detector_id": "asbPyG4BlFrvFHzloDO-",
-          "model_type": "threshold",
-          "model_id": "asbPyG4BlFrvFHzloDO-_model_threshold"
+          "detector_id" : "asbPyG4BlFrvFHzloDO-",
+          "model_type" : "threshold",
+          "model_id" : "asbPyG4BlFrvFHzloDO-_model_threshold"
         }
-      ],
-      "ad_execute_failure_count": 7
+      ]
     },
-    "y7YUQWukQEWOYbfdEq13hQ": {
-      "ad_execute_request_count": 0,
-      "ad_execute_failure_count": 0,
-      "models": []
+    "y7YUQWukQEWOYbfdEq13hQ" : {
+      "ad_execute_request_count" : 0,
+      "ad_execute_failure_count" : 0,
+      "models" : [ ]
     },
-    "cDcGNsPoRAyRMlPP1m-vZw": {
-      "ad_execute_request_count": 0,
-      "ad_execute_failure_count": 0,
-      "models": [
+    "cDcGNsPoRAyRMlPP1m-vZw" : {
+      "ad_execute_request_count" : 0,
+      "ad_execute_failure_count" : 0,
+      "models" : [
         {
-          "detector_id": "asbPyG4BlFrvFHzloDO-",
-          "model_type": "rcf",
-          "model_id": "asbPyG4BlFrvFHzloDO-_model_rcf_2"
+          "detector_id" : "asbPyG4BlFrvFHzloDO-",
```

Comment:
How about we replace all `asbPyG4BlFrvFHzloDO-` with `m4ccEnIBTXsGi3mvMt9p` in this example? As we are using this detector in other sample APIs. I have changed in public API example doc. You can copy directly.

Reply:
Done.